README.md

```diff
@@ -1,17 +1,42 @@
 # govuk-dependencies
 
-## Sinatra Application For Dependency Reporting
+A tool for viewing all of the pull requests made by Dependabot to Gov.Uk repos, as well as for
```

Comment:
Would also mention 'open pull requests' here:

'A tool for viewing all of the outstanding open pull requests made by Dependabot'...